openslides_backend/action/meeting/unset_font.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Dict
+
+from ...models.models import Meeting
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("meeting.unset_font")
+class MeetingUnsetFontAction(UpdateAction):
+    """
+    Action to unset a font form a meeting.
+    """
+
+    model = Meeting()
+    schema = DefaultSchema(Meeting()).get_default_schema(
+        title="meeting unset font schema",
+        description="An array of meeting_id and place to unset the font.",
+        additional_required_fields={
+            "meeting_id": {"type": "integer"},
```

Comment:
please use the schemas provided in `schema.py`, in this case `required_id_schema`

However, see https://github.com/OpenSlides/openslides-backend-coordination/issues/155#issuecomment-721823964: Maybe that's not necessary and we can just use the normal update schema.

Reply:
Update: See https://github.com/OpenSlides/openslides-backend-coordination/issues/155

Reply:
Used update schema, and updated payload.